src/cp/cp_control_point.c

```diff
@@ -298,12 +265,8 @@ __wt_conn_control_point_init_all(WT_SESSION_IMPL *session)
 int
 __wt_session_control_point_init_all(WT_SESSION_IMPL *session)
 {
-#if SESSION_CONTROL_POINTS_SIZE == 0
```

Comment:
Put these lines back. They are needed for proper compilation if the per-session control point is deleted.